.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
   "singleQuote": true,
   "trailingComma": "es5",
-  "printWidth": 100
+  "printWidth": 100,
+  "arrowParens": "avoid"
```

Comment:
I don't think we should change this setting.

Reply:
I agree not to change it right now, but I disagree with this decision 😄